eth2deposit/intl/fr/cli/existing_mnemonic.json

```
@@ -0,0 +1,25 @@
+{
+    "validate_mnemonic": {
+        "err_invalid_mnemonic": "Ceci n'est pas une phrase mnémonique valide, veuillez vérifier s'il existe des fautes de frappe."
+    },
+    "existing_mnemonic": {
+        "arg_existing_mnemonic": {
+            "help": "Générer (ou récupérer) des clés à partir d'une phrase mnémonique existante"
+        },
+        "arg_mnemonic": {
+            "argument": "_mnemonic",
```

Somehow French people like `_` over `-`? 😂
Changing back:

```suggestion
            "argument": "--mnemonic",
```